include/usb/class/usb_hid.h

```diff
@@ -235,6 +235,203 @@ struct hid_ops {
 }
 
 
```

Comment:
You could probably remove extra new line here

Reply:
Removed.